packages/paste-core/components/combobox/src/helpers.tsx

```diff
@@ -1,12 +1,17 @@
-import type {Item as ItemType} from './types';
+import type {Item} from './types';
```

Comment:
Issue: I think these helper functions should probably have tests.

Reply:
They do, in Combobox.spec.tsx under Unit Tests